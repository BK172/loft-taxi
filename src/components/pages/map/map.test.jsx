import React from 'react';
import renderer from 'react-test-renderer';
import Map from './map';

jest.mock(`./map`);

beforeAll(() => {
  jest.fn().mockImplementation(() => {
    return {
      remove: jest.fn()
    };
  });
});

describe('Map component test', () => {
  it('Should render Map correctly', () => {
    const tree = renderer.create(<Map />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});