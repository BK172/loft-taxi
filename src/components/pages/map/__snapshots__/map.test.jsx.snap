// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map component test Should render Map correctly 1`] = `null`;
